modules/rosa-sts-creating-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+[id="rosa-sts-creating-cluster_{context}"]
+= Creating your cluster with STS
+
+You can create an {product-title} cluster with STS using the `rosa` CLI.
+
+.Prerequisites
+
+You have completed the installation prerequisites.
+
+[NOTE]
+====
+link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Shared VPCs] are not currently supported for ROSA installs.
+====
+
+.Procedure
+
+. You can create a cluster using the default settings or by specifying custom settings using the interactive mode. To view other options when creating a cluster, enter `rosa create cluster --help`.
++
+Creating a cluster can take up to 40 minutes.
++
+[NOTE]
+====
+Multiple availability zones (AZ) are recommended for production workloads. The default is a single availability zone. Use `--help` for an example of how to set this option manually or use interactive mode to be prompted for this setting.
```

Comment:
s/ (AZ) / (Multi-AZ)

to match the abbreviation we're using elsewhere in the docs